recipes/mechanize/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "mechanize" %}
+{% set version = "0.2.5" %}
+{% set sha256 = "2e67b20d107b30c00ad814891a095048c35d9d8cb9541801cebe85684cc84766" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py3k]
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - mechanize
+
+about:
+  home: http://wwwsearch.sourceforge.net/mechanize/
+  license: BSD License or Zope Public License
```

Comment:
Which BSD license? Please add clauses e.g. `BSD 3-Clause`. Also please add the version of `Zope Public License` and please feel free to shorten to `ZPL`.